Starting a new project.md

```diff
@@ -0,0 +1,68 @@
+## Starting a new project
+
+When you're starting a new project, there are a few things you need to do:
+
+#### 1. Open a repository
+
+Ask your team lead or DevOps to open a repository.
+
+#### 2. Fill out Project setup sheet
+
+We've come up with a project setup template in cooperation with DevOps team. It's purpose is to provide the DevOps with all the details needed for deploying your application to staging and production servers.
+
+It contains general info about your project that your project manager can fill out as well as more technical information like frameworks and technologies used, build and deploy setup, third party services that need to be set up...
+
+Ask your PM for a link to the sheet template.
+
+**Fill in the blanks on Backend related stuff.**
+
+If you need any other services that aren't mentioned, **provide as many information as possible**
+
+**Project setup sheet needs to be up to date** with new changes to project setup and services used. If you're adding some new dependency or service or changing something, please update it in project setup sheet and notify DevOps to make these changes.
+
+
+#### 3. Generate new application
+
+Use [**default rails template**](https://github.com/infinum/default_rails_template/) to start your new application.
+
+Most of our work is based on the Rails framework. We have some standard configurations and gems that are basically in every Rails application. To make our lives easier, we've made a default template to generate new Rails applications that meet those standards.
```

Comment:
I'd add here: 

Please always update the default rails template if something gets broken or out of date when you're starting a new project.